Light.json

```diff
@@ -16003,7 +16003,7 @@
       ],
       "front": {
         "title": "Escape Pod & We're Doomed",
-        "imageUrl": "https://res.starwarsccg.org/cards/Virtual4-Light/hires/escapepodweredoomed.png",
```

Comment:
`Virtual4-Light/hires/escapepodweredoomed.png` does exist. We can continue to use it here.
![](https://res.starwarsccg.org/cards/Virtual4-Light/hires/escapepodweredoomed.png)

Reply:
This is the only imageUrl that uses the hires image. Every other card that has a hires png still uses the large gif url here. It seems like it would be better to be consistent.